pull_seqr_pedigrees.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import getpass
+import argparse
+
+
+def pull_project_peds(args):
+    """
+    Uses the seqr API to retrieve a list of projects' pedigrees. Writes a single pedigree,
+    'pulled_seqr_pedigrees.txt', using the retrieved pedigrees. Failing projects are written
+    to 'projects_not_pulled.txt'
+    """
+    email=args.email
+    projects=args.projects
+
+    with requests.Session() as s:
+        p = s.post(
+            "https://seqr.broadinstitute.org/api/login", 
+            json={
+                'email': email,
+                'password': getpass.getpass(),
+            }
+        )
+        final_ped = open('seqr_pedigrees.txt', 'w')
+        final_ped.write('Project_GUID\tFamily_ID\tIndividual_ID\tPaternal_ID\tMaternal_ID\tSex\n') 
```

Comment:
also a style thing - but '\t'.join(['column1', 'column2' ... ]) + "\n" reduces chance of typos

Reply:
I've converted the variable writing to this method but have to cast the paternal and maternal IDs as they are occasionally `None`. I'm not sure which I like more but went with your suggestion here.